src/theVoice/index.tsx

```diff
@@ -1,22 +1,15 @@
 import styled from 'styled-components'
-import { space, color, font, responsiveBreakpoints } from '_utils/branding'
+import { space, color, responsiveBreakpoints } from '_utils/branding'
 
 import TheVoice from './TheVoice'
 
 const StyledTheVoice = styled(TheVoice)`
-  & {
-    text-align: center;
-    font-size: ${font.xl.size};
```

Comment:
Actually just noticed that, it's in the spec, on desktop the font size should be bigger. Can you add it to the mediaLarge breakpoint?

Reply:
@wakooka  the ui-library don't have the size defined on spec on the Font SIze scale :S
should we add it anyway?